Backend/app/loan_management/manage_loans_service.py

```diff
@@ -250,24 +250,75 @@ def _check_reserved_book(
             )
         return None
 
-    def consult_limit_by_user_email(self, email: str) -> bool:
+    def cant_reserved_loans(self, email: str) -> int:
         cant_reserved_loans = self.query_database(
             """SELECT COUNT(*)
                 FROM loan
                 WHERE userEmail = ?
                 AND loanStatus = ? AND returnDate IS NULL""",
             (email, "reserved"),
         )
+        return cant_reserved_loans[0]
+
+    def cant_loaned_books(self, email: str) -> int:
```

Comment:
You could have a single query to count both reserved and loaned books, I don't think it's necessary to separate them.